deeds-dapp-service/src/main/java/io/meeds/deeds/service/MeedTokenMetricService.java

```diff
@@ -234,4 +239,11 @@ private LocalDate getTodayId() {
     return LocalDate.now(ZoneOffset.UTC);
   }
 
+  /**
+   * Sets Recent Metric to null
+   */
+  public void setRecentMetricToNull() {
+    this.recentMetric = null;
```

Comment:
this is used for Unit Tests only, please add `@Setter` in declared field to be able to use it as done for `@Getter` :
https://github.com/Meeds-io/deeds-dapp/blob/56428fe4f4d164d333307e1cb7874a285cf80bd2/deeds-dapp-service/src/main/java/io/meeds/deeds/service/MeedTokenMetricService.java#L66-L67